### PR TITLE
Allow copy and move file operations to opt-in to overwrite.

### PR DESCRIPTION
**Pull request by rdmark**
- Allow create, copy and move file operations to opt-in to overwrite
- Use this when extracting packaged properties to avoid it getting stuck in the images dir
- Better error handling in common file operation methods

### CODE REVIEW

File: python/common/src/piscsi/file_cmds.py [hunk 1]
```diff
@@ -213,16 +222,24 @@ def copy_file(self, file_path, target_path):
         parameters = {"target_path": target_path}
         if not target_path.parent.exists():
             target_path.parent.mkdir(parents=True)
-        if target_path.parent.exists() and not target_path.exists():
-            copyfile(str(file_path), str(target_path))
+        if overwrite_target or (not target_path.exists() and not overwrite_target):
```
Comment (nucleogenic) [inline python/common/src/piscsi/file_cmds.py:225]: Should be able to simplify this to `if overwrite_target or not target_path.exists()`

**Review by nucleogenic — APPROVED** [review]

File: python/common/src/piscsi/file_cmds.py (hunk 1 above)
Comment (rdmark) [reply python/common/src/piscsi/file_cmds.py]: Of course, I knew I was overcomplicating it for myself. Fixed! Also pushed a few more improvements.

**Comment by rdmark** [comment]
@Pacjunk Please let us know if this resolves your problem. I made a simple test case for myself that passed.

**Comment by Pacjunk** [comment]
@rdmark Seems to be OK. Thanks

**Pull request merged**